### PR TITLE
Change the dependencies install display

### PR DESCRIPTION
**Pull request by Jako**
### What does it do?
- Open the dependency tab, when the dependencies are not met.
- Change MODX/PHP dependencies display, since they can't be downloaded/installed at that place.
- Use less space for installed packages/available versions.

### Why is it needed?
Before:
Before
After:
After

### Related issue(s)/PR(s)
None known

### Needs Build
Due to CSS changes the the theme needs `grunt rebuild`

### CODE REVIEW

**Comment by JoshuaLuckers** [comment]
Will this also change the order when updating an extra? I would prefer to see the changelog first there.

**Comment by Jako** [comment]
The dependencies are only displayed, if the package has some. But then they are first. I will try to add a tab change, if the dependencies are not met.

**Comment by Jako** [comment]
Now the cangelog is first again and the tab is changed, when the dependencies are not met.

File: manager/assets/modext/workspace/package.panels.js [hunk 1]
```diff
@@ -35,9 +35,9 @@ MODx.panel.PackageMetaPanel = function(config) {
 };
 Ext.extend(MODx.panel.PackageMetaPanel,MODx.VerticalTabs,{
 	updatePanel: function(meta, record){
-		if(meta.changelog != undefined){
-			this.addTab(_('changelog'), 'changelog', meta);
-		}
+        if(meta.changelog != undefined){
```
Comment (JoshuaLuckers) [inline manager/assets/modext/workspace/package.panels.js:38]:
```suggestion
        if (meta.changelog !== undefined) {
```

File: manager/assets/modext/workspace/package.panels.js [hunk 2]
```diff
@@ -147,9 +147,9 @@ Ext.extend(MODx.panel.PackageBeforeInstall, MODx.panel.PackageMetaPanel,{
         Ext.getCmp('package-list-reset').show();
         installBtn.hide().signature = '';
         setupoptionsBtn.hide();
-		if(meta.changelog != undefined){
-			this.addTab(_('changelog'), 'changelog', meta);
-		}
+        if(meta.changelog != undefined){
```
Comment (JoshuaLuckers) [inline manager/assets/modext/workspace/package.panels.js:150]:
```suggestion
        if (meta.changelog !== undefined) {
```

**Review by JoshuaLuckers — COMMENTED** [review]
For a few lines the indentation has changed from tabs to spaces, since these lines have been changed it's good to add strict comparisons for these lines.

**Comment by Jako** [comment]
Now it uses less space for installed packages (requirements are met)

File: manager/assets/modext/workspace/package.panels.js (hunk 1 above)
Comment (Jako) [reply manager/assets/modext/workspace/package.panels.js:38]: Let's do that in 3.0

File: manager/assets/modext/workspace/package.panels.js (hunk 2 above)
Comment (Jako) [reply manager/assets/modext/workspace/package.panels.js:150]: Let's do that in 3.x

**Review by JoshuaLuckers — DISMISSED** [review]

**Comment by Jako** [comment]
'available' and 'not available' display value for PHP and MODX.

**Comment by Jako** [comment]
Now we 'just' need a displayname and name column in package dependencies. pdoTools is found but i.e. SEO Pro and Image+ could not be installed as dependency, see #12666 

This should be fixed with #14261 and #14260.

**Review by JoshuaLuckers — APPROVED** [review]

File: manager/assets/modext/workspace/package.panels.js (hunk 2 above)
Comment (Mark-H) [reply manager/assets/modext/workspace/package.panels.js:150]: IMO there's nothing wrong with correcting code style on things you're editing in 2.x. As these lines are already marked as changed, correcting the code style wouldn't cause extra conflicts.

**Comment by Mark-H** [comment]
What's a good extra to test this against?

**Comment by Jako** [comment]
I’ll send one by mail/slack.

**Comment by Jako** [comment]
The Crosslinks package contains PHP/MODX dependencies. If I remember right, the BlogIt package contains extras dependencies.

**Comment by ghost** [comment]
I know that Flatso has dependencies; I believe it was intentionally designed to demonstrate the use of them as well as some kind of theming system.

**Review by opengeek — APPROVED** [review]

**Pull request merged**